habitat_sim/bindings/__init__.py

```diff
@@ -7,31 +7,21 @@
 # TODO: this whole thing needs to get removed, kept just for compatibility
 #   with existing code
 
-modules = [
-    "cuda_enabled",
-    "SceneNodeType",
-    "GreedyFollowerCodes",
-    "GreedyGeodesicFollowerImpl",
-    "MultiGoalShortestPath",
-    "PathFinder",
-    "PinholeCamera",
-    "SceneGraph",
-    "SceneNode",
-    "Sensor",
-    "SensorSpec",
-    "SensorType",
-    "ShortestPath",
-    "SimulatorConfiguration",
-    "ConfigurationGroup",
-    "RigidState",
-]
-
-from habitat_sim._ext.habitat_sim_bindings import (  # noqa: F401 isort:skip
-    Simulator as SimulatorBackend,
-)
-
-exec(
-    "from habitat_sim._ext.habitat_sim_bindings import ({})".format(", ".join(modules))
+from habitat_sim._ext.habitat_sim_bindings import (
```

Comment:
What's going on here? Why did we use the `exec` before? Why don't we want to define `__all__` any more?

Reply:
I read the documentation and __all__ is implicitly set to be all imported things without an underscore, so we actually don't need it. It wasn't useful because it wasn't parseable by any IDEs anyhow.